core/lib/zksync_core/src/eth_sender/tests.rs

```diff
@@ -442,6 +448,7 @@ async fn three_scenarios() -> anyhow::Result<()> {
             .eth_sender_dal()
             .get_inflight_txs()
             .await
+            .unwrap()
```

Comment:
Nit: Since `eth_sender` tests return `anyhow::Result`, errors can be mapped here.

Reply:
The error propagation in this tests seems somewhat inconsistent (there are both unwraps and returns). I'd rather lean towards standardizing on unwraps here (since it is a test), but it is out of scope of this pr.